src/content/docs/en/reference/api-reference.mdx

```diff
@@ -880,10 +880,12 @@ import { Code } from 'astro/components';
 ---
```

Comment:
I can’t comment on the appropriate line above, but this code example would probably be easier to understand if we highlighted the extra attributes in the “optional” examples by adding metadata to the opening of the code block: https://github.com/withastro/docs/blob/main/WRITING.md#text-highlighting

Reply:
Done! I'm actually surprised how flexible the highlighter syntax is to handle regexes 😄 

<img width="691" alt="image" src="https://user-images.githubusercontent.com/34116392/235858958-3da3f74e-4228-44bf-acd7-a41733981068.png">

Reply:
We have @hippotastic to thank for that 💜